src/app/deed-model-display/deed-model-display.component.html

```diff
@@ -0,0 +1,77 @@
+<mat-accordion class="container">
+  <mat-expansion-panel>
+  <mat-expansion-panel-header class="deeds-page--title--header">
+    <mat-panel-title class="deeds-page--title--header--item">
+      {{deed.name}}
+    </mat-panel-title>
+    <mat-panel-description class="deeds-page--title--header--location">
+      <mat-icon>place</mat-icon>
+      {{deed.location}}
+    </mat-panel-description>
+    <mat-panel-description class="deeds-page--title--header--icon">
+      <mat-icon *ngIf="deed.category.name === 'Help for elderly'">supervised_user_circle</mat-icon>
+      <mat-icon *ngIf="deed.category.name === 'Help for kids'">child_care</mat-icon>
```

Comment:
consider moving string values to constants file, and reusing constant variables instead of string literals in template